docs/log-correlation.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[log-correlation]]
+== Log correlation
+
+In order to correlate logs from your app with transactions captured by the
+Elastic APM Node.js Agent, your logs must contain one or more of the following identifiers:
+
+* `transaction.id`
+* `trace.id`
+* `span.id`
+
+The Node.js Agent provides the `apm.currentTraceIds` method to help with this task.
+
+:leveloffset: -1
+include::agent-api.asciidoc[tag=currentTraceIds]
+:leveloffset: +1
+
+Using your favorite logging framework, you'd then need to inject this information into your logs.
+
+When your logs contain the appropriate identifiers, the final step is to ingest them into the same
+Elasticsearch instance that contains your APM data. See
+{apm-overview-ref-v}/observability-integrations.html#_ingest_your_logs_into_elasticsearch[Ingest your logs into Elasticsearch]
+for more information.
```

Comment:
This page uses *tagged regions* to pull the `apm.currentTraceIds` information in from the `agent-api` file. Since you can't see how it looks here, here's how it renders. I'd like some feedback on this:
<img width="689" alt="Screen Shot 2019-10-02 at 2 56 05 PM" src="https://user-images.githubusercontent.com/5618806/66085023-ef6d1e80-e524-11e9-962e-9a9ecc566ae3.png">